src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.html

```diff
@@ -96,14 +96,16 @@ <h1 id="enroll-header" class="text-break">Enroll Students for {{ courseId }}</h1
               </div>
               <div class="enroll-button-group">
                 <button
+                    [ngClass]="{'w-50': isEnrolling}"
                     type="submit"
                     title="Enroll"
                     id="btn-enroll"
                     name="button-enroll"
                     class="btn btn-success float-right"
                     [disabled]="isEnrolling"
                     (click)="submitEnrollData()">
-                  <tm-ajax-loading *ngIf="isEnrolling"></tm-ajax-loading>
+                    <tm-progress-bar *ngIf="isEnrolling"></tm-progress-bar>
+                    <tm-ajax-loading *ngIf="isEnrolling"></tm-ajax-loading>
```

Comment:
Unrelated change and inconsistent indentation (compare with the `Enroll students` line after this)